### PR TITLE
python3Packages.aiokafka: init at 0.5.2

### PR DESCRIPTION
**Pull request by costrouc**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/aiokafka/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  src = fetchFromGitHub {
+    owner = "aio-libs";
+    repo = "aiokafka";
+    rev = "v${version}";
+    sha256 = "062kqsq75fi5pbpqf2a8nxm43pxpr6bwplg6bp4nv2a68r850pki";
+  };
+
+  buildInputs = [
+    cython zlib
```
Comment (jonringer) [inline pkgs/development/python-modules/aiokafka/default.nix]: cython should go in nativeBuildInputs

Comment (costrouc) [reply pkgs/development/python-modules/aiokafka/default.nix]: fixed.

**Comment by costrouc** [comment]
@GrahamcOfBorg build python3Packages.aiokafka

**Comment by jonringer** [comment]
@GrahamcOfBorg build python3Packages.aiokafka

**Pull request merged**